svc-events/events/submittestevent.go

```diff
@@ -153,7 +154,13 @@ func validAndGenSubTestReq(reqBody []byte) (*common.Event, string, string, []int
 	if val, ok := req["EventTimestamp"]; ok {
 		switch v := val.(type) {
 		case string:
+			_, err := time.Parse(time.RFC3339, v)
+			if err != nil {
+				return nil, response.PropertyValueTypeError, "error: optional parameter EventTimestamp must be of type string", []interface{}{fmt.Sprintf("%v", v), "EventTimestamp"}
```

Comment:
error: optional parameter EventTimestamp must be of type string

can be changed to 

error: Error in Parsing time
since it is misleading as we already check the case as string.

Reply:
added a valid error message